src/locale/misc/smartSearch/en.yml

```diff
@@ -128,6 +129,34 @@ random:
     examples:
         one: Randomly add 20 people in the organization.
         two: Randomly remove 15% of the people in the organization.
+sub_query:
+    inputString: "{addRemoveSelect} people that match {query}."
+    addRemoveSelect:
+        add: Add
+        sub: Sub
+    examples:
+        one: Remove people who match Smart Search Query 'People who live in Stockholm'.
+        two: Add people who match the target group of call Assignment 'Assignment one'.
+query:
```

Comment:
Why are the following under this separate key `query` and not under the `sub_query` key? That seems to be the pattern of other filters.

Reply:
Actually taking a closer look I realize there are many cases where things are not contained under the scope in which they live in the UI. Maybe there is some reason I'm missing, or maybe this is something to handle separately in the refactor issue?

Reply:
This started when I moved timeFrame out of scope; I began to put all similar potential reusable 'snippets' in their own scope, but of course, none of them except timeFrame are actually reused. 
Added to #337

Reply:
I think this is a reasonable approach, although I would add them all under a common scope in the YAML, e.g. `misc.smartSearch.widgets` or something. But let's rivisit that for #337.